src/app/components/server-compare-charts/server-compare-charts.component.scss

```
@@ -39,6 +39,22 @@
       }
 
       scroll-margin-top: 10rem;
+
+      td {
+        vertical-align: middle;
+      }
+
+      td.compare-section-header-action-cell {
+        @apply bg-secondary;
+        align-items: center;
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)